duckduckgo/duckduckgo-privacy-extension
# Consolidate Chrome events

**Reviewer:** @jdorweiler 

## Description:

This places all the Chrome events in the same place. Hopefully that'll make it easier to make those modules testable, and do the Safari port.

This may generate a merge conflict with https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/183, which I'll be happy to resolve.

More detailed comments inline.

## Steps to test this PR:

1. The most important bit is making sure that the post-install page opens once you install
2. Messages should be easy to send (e.g. whitelisting)
3. The lists should auto-update (you may need to tweak the timing [here](https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/f790591de2b8c0e531c4f161c588b4ef5443d56f/shared/js/background/chrome-events.es6.js#L172-L174) so you don't have to wait 30 minutes)
4. HTTPS redirects / tracker blocking should work

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
